### DISCUSSION TITLE
曹粉是什么粉……？

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#679] [General]
```
@传说中的养乐多 今日的推荐菜单是：
早餐：曹粉
午餐：金枪鱼三明治
晚餐：鳗鱼饭
夜宵：甜甜圈
```

**Comment by Arondight** [comment]
我也不知道，菜单目前来源有两部分

1. 一万年前我在百度上随便搜的一份菜单粘贴复制了过来
2. 老大哥加的满汉全席 @Mark9804

**Comment by mark9804** [comment]
曹粉不是我加的，我们群里当时也进行了一场激烈的百度

要是完全没人知道是什么的话就删掉吧

**Reply by Arondight** [reply]
我感觉也是，意义不明的都删了吧

**Reply by mark9804** [reply]
我来删吧，老大哥有没有什么要补充的，我加的菜单是一个妹子给我的，健康过头了

**Reply by Arondight** [reply]
没有……我一直把这个功能当作抽奖测运气之类的功能的，具体吃啥真的不关心

**Reply by mark9804** [reply]
行，我群里真的会用这个决定吃什么……

**Reply by Xm798** [reply]
> 我来删吧，老大哥有没有什么要补充的，我加的菜单是一个妹子给我的，健康过头了

羡慕妹子

（另外，回邮件是没有用的吗

**Reply by Xm798** [reply]
> 行，我群里真的会用这个决定吃什么……

好多外卖都点不到 嗨呀

**Reply by mark9804** [reply]
直接回邮件应该是有用的

**Reply by Xm798** [reply]
> 直接回邮件应该是有用的
> 
> 
> 
> > **Reply to this email directly**, view it on GitHub, or unsubscribe.
> 
> You are receiving this because you were mentioned.

上个也回了个邮件，但似乎没甚魔卵用
<details>

![image](https://user-images.githubusercontent.com/48074861/156140134-5515a8f8-5ed5-4c77-9fa3-2e5fded4c39f.png)
</details>

**Reply by mark9804** [reply]
那我不太清楚为什么没有用

另外 @Arondight 老大哥上次是怎么给 menu.yml 里面的项目排序的，有轮子或者插件方便分享吗

**Reply by Arondight** [reply]
可以用 js-yaml，已经装好了，load 排序 dump，我用的 vim 自带的排序，因为 dump 把注释丢了，vim 方便一些吧

**Reply by Arondight** [reply]
`:10,100sort`

**Reply by mark9804** [reply]
我去，vim 真好用啊

**Reply by Arondight** [reply]
> 我去，vim 真好用啊

算是个合格的编辑器吧，工作七八年全都是 vim 写的代码，ide 的工作用插件和系统命令辅助的
